modules/@apostrophecms/widget-type/index.js

```diff
@@ -100,7 +100,11 @@ module.exports = {
     neverLoadSelf: true,
     initialModal: true,
     placeholder: false,
-    placeholderClass: 'apos-placeholder'
+    placeholderClass: 'apos-placeholder',
+    // two-thirds, half or full:
```

Comment:
`one-half` in the ticket but `half` already exists in the current implementation.